### PR TITLE
Update: Page 支持排版

### PR DESCRIPTION
**Pull request by chenjiandongx**
本次 PR 内容，

1. Page 支持排版

### CODE REVIEW

File: pyecharts/charts/composite_charts/page.py [hunk 1]
```diff
@@ -34,6 +41,16 @@ def add(self, *charts):
                 self.js_dependencies.add(d)
         return self
 
+    @staticmethod
```
Comment (chfw) [inline pyecharts/charts/composite_charts/page.py]: Protected method as static? Strange feeling!

Comment (chenjiandongx) [reply pyecharts/charts/composite_charts/page.py]: 确实奇怪 我等下改正过来

**Comment by chenjiandongx** [comment]
@chfw please review again

**Comment by chfw** [comment]
```
Code without tests is bad code

        -- Michael feathers, <<Working Effectively with Legacy Code>>
```

**Comment by chenjiandongx** [comment]
@chfw Code with tests is good code

**Pull request merged**